# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wpa_supplicant/install.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- wpa_supplicant-0.5.7/Makefile.vanilla	2007-03-19 22:58:11.000000000 +0100
+++ wpa_supplicant-0.5.7/Makefile	2007-03-19 22:59:44.000000000 +0100
@@ -11,9 +11,9 @@
 # Include directories for CVS version
 CFLAGS += -I. -I../utils -I../hostapd
 
-ALL=wpa_supplicant wpa_passphrase wpa_cli dynamic_eap_methods
+ALL=wpa_supplicant wpa_passphrase wpa_cli
 
-all: verify_config $(ALL)
+all: verify_config dynamic_eap_methods $(ALL)
 
 verify_config:
 	@if [ ! -r .config ]; then \
